src/main/java/command/DeadlineCommand.java

```diff
@@ -0,0 +1,62 @@
+package command;
+
+import duke.DukeException;
+import duke.Ui;
+
+import task.Deadline;
+import task.TaskList;
+
+/**
+ * This class will carry the implementation of the Deadline Command
+ * Takes in the command line command,parses as well as executing it.
+ *
+ * @author Bryan Ong
+ *
+ */
+public class DeadlineCommand {
+    private String[] inputs;
+    private static StringBuilder strBuild = new StringBuilder();
+
+    public DeadlineCommand(String[] inputs) {
+        this.inputs = inputs;
+    }
+
+
+    /**
+     * This method is used to create the Deadline command.
+     * Parsing as well as creation of Deadline object is done here.
+     */
+    public String create() throws DukeException {
+        assert inputs[0].equalsIgnoreCase("DEADLINE") : "Contact developer on Deadline bug";
+        if (inputs.length == 1) {
+            throw new DukeException(Ui.printMissingName());
+        } else {
+            boolean isName = true;
```

Comment:
It might be possible to prevent nested code by having the else block be part of the main code, instead of an else clause.